docs/sources/panels/field-options/standard-field-options.md

```diff
@@ -86,6 +87,20 @@ You can also paste a native emoji in the unit picker and pick it as a custom uni
 
 Grafana can sometime be too aggressive in parsing strings and displaying them as numbers. To make Grafana show the original string create a field override and add a unit property with the `string` unit.
 
+## Color
+
+> **Note:** Only available in Grafana 7.3+.
+
+The field color option defines how Grafana colors series or fields. There are multiple modes here that work very differently, and their utility depends largely on what visualization you currently have selected.
+
+Continuous color modes use the percentage of a value relative to min and max to interpolate a color.
+
+- **Fixed color:** Set a specific color using the color picker. Mostly useful from an override rule.
+- **Classic:** Color is derived from the series or field index and looking up a corresponding color in the classic color palette. Useful for Graphs.
+- **Vibrant:** Color is derived from the series or field index and looking up a corresponding color in the vibrant color palette. Useful for Graphs.
```

Comment:
I guess in the future we will introduce some more palettes. I suggest having these options prefixed in the UI with `Palette > Name of the palette`. Same goes with spectrum/continuous.

In the UI I suggest the following order:
```
Fixed color
From thresholds
Palette > ...
Palette > ...
Continuous > ...
Continuous > ...
...
```